iphone/Classes/MediaModule.m

```diff
@@ -29,14 +29,11 @@
 #import <MobileCoreServices/UTCoreTypes.h>
 #import <QuartzCore/QuartzCore.h>
 #import <AVFoundation/AVFoundation.h>
-
 #import <UIKit/UIPopoverController.h>
-
-#if IS_XCODE_7_1
 #import <Photos/Photos.h>
 #import "TiUIiOSLivePhoto.h"
```

Comment:
The ticket is for minimum Xcode 7.0 , so should we be removing this , seeing as livePhotView is iOS 9.1 onwards not iOS 9.0.